w09h01/test/pgdp/pingutrip/ReadWayPointsTest.java

```diff
@@ -22,7 +22,14 @@ void testReturnsValuesFromFile() {
             assertEquals(expected[i], actual[i], "The value at index " + i + " is wrong");
         }
     }
-
+    
+    /* Comments:
+    Mac OS may throw IOException lazyly if the directory does not exist.
+    So if you have dealt with IOException on Mac, you still might encounter it.
+    However, this JUnit test will not be executed in Artemis.
+    So if you are sure that you have dealt with IOException, you can ignore this test.
+    (for MacOS) By: BSLK11
+    */
```

Comment:
> However, this JUnit test will not be executed in Artemis.

Do you know how Artemis is testing this sentence from the task "Sollte beim Einlesen ein Fehler auftreten, soll ein leerer Stream zurückgegeben werden."`

Reply:
No. I had this fed back to the professor and he told me to ignore this case, because his Mac also encountered this problem. But he only told that this empty string case would not be tested. However, I have no clue how they test the result.

Reply:
Because I intended to test a case where we have an error when reading the file (file can not be found)

Reply:
Maybe we can pass in the test `null` instead of an empty string?

Reply:
It will throw other Exception when the input is null, rather than IOException. I would recommend using illegal string rather than empty string, for example "A" or "你好“, and these two run properly in my IDE.

Reply:
https://github.com/MaximilianAnzinger/pgdp2223-tests/pull/203 is a better solution right?